prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -535,8 +529,18 @@ ssize_t efa_rdm_ope_prepare_to_post_send(struct efa_rdm_ope *ope,
 		 */
 		single_pkt_entry_data_size = (total_pkt_entry_data_size - 1) / *pkt_entry_cnt + 1;
 
-		/* each packet must be aligned */
+		/*
+		 * Each packet must be aligned.
+		 * single_pkt_entry_data_size & ~(memory_alignment - 1) is 0 when total_pkt_entry_data_size
```

Comment:
> single_pkt_entry_data_size & ~(memory_alignment - 1) is 0 when total_pkt_entry_data_size is smaller than memory_alignment

I think this is a typo `single_pkt_entry_data_size & ~(memory_alignment - 1) is 0 when single_pkt_entry_data_size is smaller than memory_alignment`

Then this comment does not help... I think the assertion on L542 should be sufficient.

Reply:
Thanks, removed the unnecessary comment